### PR TITLE
[SDPA-6026] Unable to easily search and find internal links

### PR DESCRIPTION
**Pull request by Jeannie-deloitte**
**JIRA**
https://digital-vic.atlassian.net/browse/SDPAP-6026

**Test Environment**
https://nginx-php.pr-89.content-reference-sdp-vic-gov-au.sdp2.sdp.vic.gov.au/

**Changes**
1. Adds editor.editor.admin_text.yml file to config/install
2. Adds editor.editor.rich_text.yml file to config/install
3. Adds editor.editor.summary_text.yml file to config/install
4. Update script in tide_core.install file to update linkit profile and filter weights to avoid editor errors

**Screenshots**
Below are screenshots of the configuration of all 3 editor settings using ckeditor toolbar located at admin/config/content/formats showing profile selected and filter order.

Screen Shot 2022-07-26 at 1 33 47 pm
Screen Shot 2022-07-26 at 1 34 02 pm
Screen Shot 2022-07-26 at 1 34 20 pm

### CODE REVIEW

File: tide_core.install [hunk 1]
```diff
@@ -1446,3 +1446,48 @@ function tide_core_update_8061() {
...
+  foreach ($filter_ids as $filter_id) {
+    $filter = $config_factory->getEditable($filter_id);
+    if ($filter->isNew()) {
+      continue;
+    }
+
+    $filter->set('filters.filter_align.weight', '-50');
+    $filter->set('filters.filter_caption.weight', '-49');
+    $filter->set('filters.linkit.weight', '-48');
+    $filter->save();
```
Comment (MdNadimHossain) [inline tide_core.install:1491]: Hi @Jeannie-deloitte , I can only see the above update hook only updating the weights but not the `allowed HTML list`. Unless I am missing something. 
I can see [allowed_html](https://github.com/dpc-sdp/tide_core/pull/331/files#diff-bfaac244bb002238402fbeca2505a522242ef93e3a9415e21ace38579cbb725dR51), https://github.com/dpc-sdp/tide_core/pull/331/files#diff-bfaac244bb002238402fbeca2505a522242ef93e3a9415e21ace38579cbb725dR99 was updated but not added in the update hook. So shouldn't those be also added to the update hook

**Review by MdNadimHossain — CHANGES_REQUESTED** [review]

File: tide_core.install (hunk 1 above)
Comment (Jeannie-deloitte) [reply tide_core.install:1491]: @MdNadimHossain Thanks for pointing that out. I didnt actually change the allowed HTML list (line 1475 is an accidental comment left in from copying another update hook). The config file upon export from Drupal has rearranged the Allowed HTML list however the actual list hasnt changed. I will update the comment to be less confusing

**Comment by MdNadimHossain** [comment]
Small lint issue @Jeannie-deloitte :)

**Comment by Jeannie-deloitte** [comment]
@MdNadimHossain Sorry about that. Should be fine now.

**Review by MdNadimHossain — APPROVED** [review]

**Pull request merged**